src/doctest.jl

```diff
@@ -17,21 +17,24 @@ will skip doctesting of manual pages altogether.
 
 Additional keywords are passed on to the main [`doctest`](@ref) method.
 """
-function doctest(package::Module; manual=true, testset=nothing, kwargs...)
+function doctest(package::Module; manual = true, testset = nothing, kwargs...)
     if pathof(package) === nothing
         throw(ArgumentError("$(package) is not a top-level package module."))
     end
     source = nothing
     if manual === true
-         source = normpath(joinpath(dirname(pathof(package)), "..", "docs", "src"))
-         isdir(source) || throw(ArgumentError("""
-         Package $(package) does not have a documentation source directory at standard location.
-         Searched at: $(source)
-         If ...
-         """))
+        source = normpath(joinpath(dirname(pathof(package)), "..", "docs", "src"))
+        if !isdir(source)
+            msg = """
+            Package $(package) does not have a documentation source directory at standard location.
+            Searched at: $(source)
+            If ...
+            """
+            throw(ArgumentError(msg))
+        end
```

Comment:
This is a manual rewrite? It's more readable, so no objections.

Reply:
Yea it is manual, formatter does
```
throw(
    ArgumentError(
        """
        Package $(package) does not have a documentation source directory at standard location.
        Searched at: $(source)
        If ...
        """
    )
)
```